### ISSUE TITLE
Text on signs is not visible using Silders Vibrant Shaders v1.281 Extreme-VL

### ISSUE THREAD

**Issue by redstonefreak589** [IrisShaders/Iris#842]
When using Silders Vibrant Shaders v1.281 Extreme-VL, there are some graphical glitches; One of them being the outlines of blocks are in a rainbow, but that's not what this issue raises.

Text on signs is invisible while using this Shader, as seen in the [screenshot here](https://imgur.com/DCeRE5m) (compared to [vanilla here](https://imgur.com/Q4pE8bR)). Apologies for the particle effects.

Using a different shader, such as BSL v8.1, it's fine ([screenshot here](https://imgur.com/2Sp2cIL))

**Comment by coderbot16** [comment]
Please do fill out the issue template - in this case, we don't immediately know what version of Minecraft you are using as well as what version of Iris you are using, which are important things to know before we can move forward with debugging.

**Comment by redstonefreak589** [comment]
My apologies, submitting a new issue now and closing this one

**Comment by coderbot16** [comment]
Thanks!
